src/Types/Input/Scalar/ScalarInputType.hack

```diff
@@ -0,0 +1,10 @@
+namespace Slack\GraphQL\Types;
+use namespace Slack\GraphQL;
+
+abstract class ScalarInputType extends NamedInputType {
```

Comment:
Should we call this `LeafInputType` to mirror `Output/LeafType.hack`?

Reply:
leaf types are scalars + enums